### PR TITLE
Remove build-time dependency on the python3-nose package

### PR DESCRIPTION
**Pull request by VladimirSlavik**
We have been using the version distributed via pip for some months already.

### CODE REVIEW

File: anaconda.spec.in [hunk 1]
```diff
@@ -58,7 +58,6 @@ BuildRequires: make
 BuildRequires: pango-devel
 BuildRequires: python3-kickstart >= %{pykickstartver}
```
Comment (poncovka) [inline anaconda.spec.in:59]: I think that this used to be a dependency of the nose tests, so we could remove it as well.

**Review by poncovka — APPROVED** [review]
Looks good to me. Thanks!

**Review by rvykydal — APPROVED** [review]
Looks good to me.

File: anaconda.spec.in (hunk 1 above)
Comment (VladimirSlavik) [reply anaconda.spec.in:59]: All tests seem to pass with it locally, let's see.

**Review by poncovka — APPROVED** [review]

**Comment by VladimirSlavik** [comment]
Squashed the two commits. I wonder about the ELN tests, I think they failed earlier only because of pykickstart.

**Review by M4rtinK — APPROVED** [review]
Looks good to me as well.

**Pull request merged**